examples/cpp/vcproj/svm_two_class_thunder_dense_batch/svm_two_class_thunder_dense_batch.vcxproj.user

```diff
@@ -1,115 +1,115 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug.static.threaded|Win32'">
-    <LocalDebuggerEnvironment>PATH=$(SolutionDir)..\..\redist\ia32;$(SolutionDir)..\..\..\..\tbb\latest\redist\ia32\vc_mt;$(PATH)</LocalDebuggerEnvironment>
+    <LocalDebuggerEnvironment>PATH=$(ProjectDir)..\..\..\..\..\redist\ia32_win\tbb\vc_mt;$(ProjectDir)..\..\..\..\..\redist\ia32_win\daal;%PATH%</LocalDebuggerEnvironment>
     <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
     <LocalDebuggerCommandArguments>
     </LocalDebuggerCommandArguments>
     <LocalDebuggerWorkingDirectory>$(ProjectDir)..\..</LocalDebuggerWorkingDirectory>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug.static.threaded|x64'">
-    <LocalDebuggerEnvironment>PATH=$(SolutionDir)..\..\redist\intel64;$(SolutionDir)..\..\..\..\tbb\latest\redist\intel64\vc_mt;$(PATH)</LocalDebuggerEnvironment>
+    <LocalDebuggerEnvironment>PATH=$(ProjectDir)..\..\..\..\..\redist\intel64_win\tbb\vc_mt;$(ProjectDir)..\..\..\..\..\redist\intel64_win\daal;%PATH%</LocalDebuggerEnvironment>
```

Comment:
why it's applicable only for svm vcxproj files?

Reply:
Because these files were merged from the master, where are the other paths.